docs/cluster_spec.md

```diff
@@ -194,6 +194,16 @@ NOTE: Where the corresponding configuration value can be empty, fields can be se
 
 Will result in the flag `--resolv-conf=` being built.
 
+#### Enable Custom metrics support
+To use custom metrics in kubernetes we have to set flag  `ENABLE_CUSTOM_METRICS` to `true` on all the kubelets.
+we can specify that in the `kubelet` spec in our cluster.yml.
```

Comment:
Maybe we should add a link to the custom metrics docs on kubernetes.io? 

https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-custom-metrics

Reply:
@mikesplain Yes, Have included that link. Thanks.